components/gitpod-db/src/project-db.spec.db.ts

```diff
@@ -30,9 +29,7 @@ class ProjectDBSpec {
 
     async wipeRepo() {
         const typeorm = testContainer.get<TypeORM>(TypeORM);
-        const manager = await typeorm.getConnection();
-        await manager.getRepository(DBProject).delete({});
-        await manager.getRepository(DBUser).delete({});
+        await resetDB(typeorm);
```

Comment:
Extracting all post-test DB cleanups to one method that ensures we don't delete needed state (e.g. admin user)